Makefile

```diff
@@ -52,22 +67,53 @@ endif
 # List of smoke test steps.  This is used by CI
 #
 
-smoke-test:	test-clean-up	build_test
-	$(TEST_BIN) smoke --spu ${DEFAULT_SPU} --local ${TEST_LOG} -r ${REPL} ${SKIP_CHECK} ${EXTRA_ARG} -- --producer-iteration=${DEFAULT_ITERATION}
-
-smoke-test-stream:	test-clean-up	build_test
-	$(TEST_BIN) smoke --spu ${DEFAULT_SPU} --local ${TEST_LOG} ${SKIP_CHECK} -- --consumer-wait=true --producer-iteration=${DEFAULT_ITERATION}
-
-smoke-test-tls:	test-clean-up build_test
-	$(TEST_BIN) smoke --spu ${DEFAULT_SPU} --tls --local ${TEST_LOG} ${SKIP_CHECK} -- --producer-iteration=${DEFAULT_ITERATION}
-
-smoke-test-tls-policy:	test-clean-up build_test
-	AUTH_POLICY=$(SC_AUTH_CONFIG)/policy.json X509_AUTH_SCOPES=$(SC_AUTH_CONFIG)/scopes.json  \
-	FLV_SPU_DELAY=$(SPU_DELAY) \
-	$(TEST_BIN) smoke --spu ${DEFAULT_SPU} --tls --local ${TEST_LOG} ${SKIP_CHECK} --keep-cluster -- --producer-iteration=${DEFAULT_ITERATION}
+smoke-test-base: test-setup smoke-test-base-nobuild
+smoke-test-base-nobuild: test-setup-nobuild
```

Comment:
can we shorten this target name?

Reply:
Do you mean shorten `smoke-test-base` or `smoke-test-base-nobuild`?

Reply:
smoke-test-base-nobuild

Reply:
I changed `smoke-test` to `smoke-test-local` and I changed `smoke-test-base` to `smoke-test`